protocol/app/app.go

```diff
@@ -602,14 +603,14 @@ func New(
 			app.Server.ExpectLiquidationsDaemon(
 				daemonservertypes.MaximumAcceptableUpdateDelay(daemonFlags.Liquidation.LoopDelayMs),
 			)
+			app.LiquidationsClient = liquidationclient.NewClient(logger)
 			go func() {
```

Comment:
As a follow up PR, can we refactor so that `go func()` all live within `*Client.Start` or live in `app.go`?

I think we currently have a mixed bag of implementation between prices vs liq/bridge daemons

Reply:
https://linear.app/dydx/issue/CORE-756/refactor-pricefeed-client-to-start-in-a-consistent-pattern-with-other